Build errors

Some errors with gcc 7.1.1:
```bash
src/tests/stdio/printf_test.c: Function ”__test_case_at_line_113”:
src/tests/stdio/printf_test.c:122:23: error: ”snprintf” output truncated before the last format character [-Werror=format-truncation=]
  test_assert_equal(4, snprintf(dest, 4, "1234"));
...
src/tests/stdio/printf_test.c:125:23: error: ”12345678” directive output truncated writing 8 bytes into a region of size 2 [-Werror=format-truncation=]
  test_assert_equal(8, snprintf(dest, 2, "12345678"));
```
Template: x86/qemu

As strange as it may sound, it's caused by my locale (fi_FI.UTF-8). But it works with ru_RU.UTF-8. I don't really understand why.
```bash
$ echo $LANG
fi_FI.UTF-8
$ make distclean
$ make confload-x86/qemu
$ make
...
In file included from ./src/include/embox/test.h:32:0,
                 from src/tests/stdio/printf_test.c:10:
src/tests/stdio/printf_test.c: Funktio ”__test_case_at_line_113”:
src/tests/stdio/printf_test.c:122:23: virhe: ”snprintf” output truncated before the last format character [-Werror=format-truncation=]
  test_assert_equal(4, snprintf(dest, 4, "1234"));
./src/include/framework/test/assert.h:129:39: huom: in definition of macro ”test_assert_equal”
  __test_assertion_handle((actual) == (expected), \
                                       ^~~~~~~~
src/tests/stdio/printf_test.c:122:23: huom: ”snprintf” output 5 bytes into a destination of size 4
  test_assert_equal(4, snprintf(dest, 4, "1234"));
./src/include/framework/test/assert.h:129:39: huom: in definition of macro ”test_assert_equal”
  __test_assertion_handle((actual) == (expected), \
                                       ^~~~~~~~
src/tests/stdio/printf_test.c:125:23: virhe: ”12345678” directive output truncated writing 8 bytes into a region of size 2 [-Werror=format-truncation=]
  test_assert_equal(8, snprintf(dest, 2, "12345678"));
./src/include/framework/test/assert.h:129:39: huom: in definition of macro ”test_assert_equal”
  __test_assertion_handle((actual) == (expected), \
                                       ^~~~~~~~
src/tests/stdio/printf_test.c:125:23: huom: ”snprintf” output 9 bytes into a destination of size 2
  test_assert_equal(8, snprintf(dest, 2, "12345678"));
./src/include/framework/test/assert.h:129:39: huom: in definition of macro ”test_assert_equal”
  __test_assertion_handle((actual) == (expected), \
                                       ^~~~~~~~
cc1: all warnings being treated as errors
make[4]: *** [mk/image_lib.mk:33: build/base/obj/src/tests/stdio/printf_test.o] Virhe 1
make[3]: *** [mk/build.mk:23: build] Virhe 2
make[2]: *** [mk/load.mk:41: build] Virhe 2
make[1]: *** [mk/main.mk:30: build] Virhe 2
make: *** [Makefile:37: all] Virhe 2
$ LANG=en make
...
Build complete
$ LANG=ru make

...
Build complete
```

--- a/mk/flags.mk
+++ b/mk/flags.mk
@@ -203,7 +203,7 @@ else
 	# application, we decided to keep explicit null checks and disable the warning.
 	override COMMON_CCFLAGS += -Wno-nonnull-compare
 
-	override GCC_VERSION := $(word 3,$(shell $(CC) -v 2>&1 | grep -e "^gcc"))
+	override GCC_VERSION := $(word 3,$(shell $(CC) --version 2>&1 | grep -e "^gcc"))
 	override GCC_VERSION_MAJOR := $(word 1,$(subst ., ,$(GCC_VERSION)))
 
 ifeq ($(GCC_VERSION_MAJOR),7)

